client/dist/styles.css

```diff
@@ -160,75 +218,90 @@ StarRatingCSS */
 /*
 StyleSelector */
 .styleRow {
+  width: 80%;
   display: flex;
-}
-.selected {
-  opacity: 0.3;
+  justify-content: center;
 }
 .checkmark {
-  position: relative;
-  left: 35%;
+  position: absolute;
+  left: 30%;
   top: 50%;
 }
 .style, .styleContainer {
-  width: 80px;
-  height: 80px;
+  position: relative;
+  width: 8vh;
+  height: 8vh;
+  margin: auto;
   border-radius: 50%;
   background-color: white;
 }
-.style {
-  position: absolute;
-  transition: .3s ease;
+#CurrentStyle {
+  font-size: 4em;
+  margin: 0;
 }
-.styleContainer {
-  display: inline-flex;
+.style {
+   transition: .3s ease;
 }
-.style:hover {
+
+.style:hover, .selected {
   opacity: 0.3;
 }
-
-#SizeSelector{
-  height: 35px;
- }
+.cartButtonContainer, #overviewSelectors {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 100%;
+}
+#CartButton {
+  border: transparent;
+  background-color: red;
+  height: 5vh;
+  width: 25vw;
+  font-size: 3em;
+  font-weight: bold;
+  border-radius: 10px;
+}
+#CartButton:hover {
+  background-color: rgb(250, 92, 92);
+}
 /*
 Overview Structure */
 #Image {
   float: left; /*It isnt though*/
   margin: auto;
   width: auto;
   display: inline-block;
-  margin-right: 20px;
+  margin-right: 50px;
 }
-#ProductInfo {
-  margin: 0;
-  margin: auto;
-  position: flex;
-  float: left;
-  width: auto;
-  display: flex;
-  flex-direction: column;
+#ProductInfo, #StyleSelector, #AddToCart {
+  min-width: 100%;
+  min-height: 33.33%;
+  max-height: 33.33%;
+  padding: 2em;
 }
 #StyleSelector {
-  margin: 0;
-  position: flex;
-  float: left;
-  width: auto;
   display: flex;
-  flex-direction: column;
+  flex-wrap: wrap;
+  justify-content: center;
 }
 #AddToCart {
-  margin: auto;
-  position: flex;
-  float: left;
-  width: auto;
   display: flex;
+  flex-wrap: wrap;
   flex-direction: column;
+  justify-content: center;
 }
 #overview {
+  display: flex;
+}
+#overviewContainer {
   margin: auto;
 }
-.leftSide {
-  float: left;
+#overviewContainer * {
+  box-sizing: border-box;
+}
+#ProductDetails {
+  display: flex;
+  margin: 10px, 0px;
```

Comment:
Another spot with px instead of relative units. Also, I don't think comma is needed when listing values for margin.